tests/test_store.py

```diff
@@ -517,6 +517,7 @@ def test_add_entity_id_and_filename_without_component_col(self, synapse_store):
             )
             assert_frame_equal(manifest_to_return, expected_df)
 
+    @pytest.mark.asyncio
```

Comment:
You actually don't need to manually mark tests as async. Add a similar `pytest.ini` file to the root of the repo and it will happen automagically.

https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/pytest.ini#L3-L4

Reply:
@BryanFauble that's great news, thank you